python/requirements.txt

```diff
@@ -40,11 +40,11 @@ kopf; python_version >= '3.7'
 scipy
 aiohttp>=3.7
 fastapi
-gym<0.24.0,>=0.21.0
+gym<0.24.1,>=0.21.0
 opencensus
 fsspec
 dm_tree
-matplotlib!=3.4.3
+matplotlib!=3.4.3; platform_system != "Windows"
```

Comment:
Do we need to make the corresponding changes in setup.py?

Reply:
No, this should be very CI specific - the CI machines don't have windows fonts installed and thus the environment variable lookup fails.

Reply:
(a regular windows install will have fonts installed)

Reply:
Got it! How about for the gym change?

Reply:
So to be clear, for windows we don't want to install matplotlib at all

Reply:
Yep sounds good! What about the change to gym?